### ISSUE TITLE
unknown command errors

### ISSUE THREAD

**Issue by anildigital** [cmacrae/spacebar#60]
For `spacebar 1.2.1` and `yabai-v3.3.7` getting following errors for `tail -f /usr/local/var/log/spacebar/spacebar.err.log`

```
unknown command 'right_shell_command' for domain 'config'
unknown command 'title' for domain 'config'
unknown command 'spaces' for domain 'config'
unknown command 'clock' for domain 'config'
unknown command 'power' for domain 'config'
unknown command 'padding_left' for domain 'config'
unknown command 'padding_right' for domain 'config'
unknown command 'right_shell' for domain 'config'
unknown command 'right_shell_icon' for domain 'config'
unknown command 'right_shell_command' for domain 'config'
```

**Comment by code-scan** [comment]
> For `spacebar 1.2.1` and `yabai-v3.3.7` getting following errors for `tail -f /usr/local/var/log/spacebar/spacebar.err.log`
> 
> ```
> unknown command 'right_shell_command' for domain 'config'
> unknown command 'title' for domain 'config'
> unknown command 'spaces' for domain 'config'
> unknown command 'clock' for domain 'config'
> unknown command 'power' for domain 'config'
> unknown command 'padding_left' for domain 'config'
> unknown command 'padding_right' for domain 'config'
> unknown command 'right_shell' for domain 'config'
> unknown command 'right_shell_icon' for domain 'config'
> unknown command 'right_shell_command' for domain 'config'
> ```

I had the same problem, but I recompiled the latest one and replaced 

```bash
➜  1.2 git:(master) ✗ make
rm -rf ./bin
mkdir -p ./bin
clang src/manifest.m -std=c99 -Wall -DDEBUG -g -O0 -fvisibility=hidden -mmacosx-version-min=10.13 -F/System/Library/PrivateFrameworks -framework Carbon -framework Cocoa -framework CoreServices -framework SkyLight -framework ScriptingBridge -framework IOKit -o bin/spacebar
➜  1.2 git:(master) ✗ bin/spacebar
^C
➜  1.2 git:(master) ✗ where spacebar
/usr/local/bin/spacebar
/usr/local/bin/spacebar
➜  1.2 git:(master) ✗ cp bin/spacebar /usr/local/bin/spacebar
cp: /usr/local/bin/spacebar: Permission denied
➜  1.2 git:(master) ✗ chmod cp bin/spacebar /usr/local/bin/spacebar
chmod: Invalid file mode: cp
➜  1.2 git:(master) ✗ sudo cp bin/spacebar /usr/local/bin/spacebar

```

The repair was successful

**Comment by muhammad-saleh** [comment]
@code-scan
I followed your steps and when I try to run spacebar it crashes, I get:
`[1]    26715 segmentation fault  ./spacebar --verbose`
Do you please know how to fix this?

**Comment by code-scan** [comment]
> @code-scan
> I followed your steps and when I try to run spacebar it crashes, I get:
> `[1] 26715 segmentation fault ./spacebar --verbose`
> Do you please know how to fix this?

don't use *left_shell_icon* and *right_shell_icon* option
```
spacebar -m config left_shell on
# spacebar -m config left_shell_icon 1
spacebar -m config left_shell_command "~/.config/spacebar/bar.sh"

spacebar -m config right_shell          on
# spacebar -m config right_shell_icon     1
spacebar -m config right_shell_command  "echo -n Mode: ; yabai -m query --spaces --space | jq -r '.type'"

```

**Comment by cmacrae** [comment]
Sorry everyone, this is a failure on my part to make the documentation consistent with the release version. This isn't the first time this has come up, so I'll admit it's an oversight on my part.

This project's `master` resources should be considered unstable. When I'm making changes getting ready to release a block of features, I do so on their own branches, then merge into master ahead of time. Included in these changes are usually updates to the documentation/example configuration.

This of course means, if you're using a release that doesn't have these features yet, the examples won't work.

So, in short, the "fix" for this issue is to either use spacebar built from the HEAD of master, or, use the example config/documentation from the release tag (for example, [here's 1.2.1](https://github.com/cmacrae/spacebar/tree/v1.2.1))

That said, I appreciate how confusing this is, so I'll open an issue and think about how I can better manage this going forward :)

**Comment by cmacrae** [comment]
Closing this in favour of the issue created above ☝️
